org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/backend/filebased/GitBasedRepository.java

```diff
@@ -56,30 +57,34 @@
 	public GitBasedRepository(String repositoryLocation) throws IOException, NoWorkTreeException, GitAPIException {
 		super(repositoryLocation);
 		FileRepositoryBuilder builder = new FileRepositoryBuilder();
-		Repository gitRepo = builder.setWorkTree(this.repositoryRoot.toFile()).setMustExist(false).build();
-		if (!new File(this.determineRepositoryPath(repositoryLocation) + "/.git").exists()) {
-		    gitRepo.create();
+		this.gitRepo = builder.setWorkTree(this.repositoryRoot.toFile()).setMustExist(true).build();
```

Comment:
Why is `true` passed to `setMustExist`? What happens if the repository does not exist? I thought that `false` ensures that the method succeeds and the next three lines will generate the repository.